test/unit/package_manager_base_test.py

```diff
@@ -14,6 +14,9 @@ def setup(self):
         repository.root_dir = 'root-dir'
         self.manager = PackageManagerBase(repository)
 
+    def teardown(self):
+        del self.manager
```

Comment:
Actually, this should not be used. Usage of `del` is a code smell. Like very stinky smell, actually. This op should be used only preventing circular references, like in case of acessing traceback frame via variable and some very rare cases when typical GC is not enough (high-density memory calculations etc).

But in your case it is clearly code smell. I would revisit the design.